MMA Removal Utility/Prerequisites.md

```diff
@@ -0,0 +1,67 @@
+## **Prerequisites**
+
+### **Validate prerequisites on machine**  
+<br/>
+
+  1.  Installation steps are supported using following OS options: 	
+
+      - Windows 10 or higher
+      - Windows Server 2019 or higher
+  
+  <br/>
+
+  2. PowerShell 5.0 or higher:
+
+      All setup steps will be performed with the help of PowerShell ISE console. If you are unaware of PowerShell ISE, refer [link](PowerShellTips.md) to get a basic understanding.
+      Ensure that you are using Windows OS and have PowerShell version 5.0 or higher by typing **$PSVersionTable** in the PowerShell ISE console window and looking at the PSVersion in the output as shown below.) 
+      <br/>
+
+      If the PSVersion is older than 5.0, update PowerShell from [here](https://www.microsoft.com/en-us/download/details.aspx?id=54616). 
+      
+        ![PowerShell Version](../Images/00_PS_Version.png)  
+
+  <br/>
+
+  3. PowerShell language mode FullLanguage:  
+
+      To run AzTS MMA Removal Utility setup script, PowerShell language mode for the session must be FullLanguage.
+      Ensure that you are using FullLanguage mode by typing **$ExecutionContext.SessionState.LanguageMode** in the PowerShell ISE console window. More details about PowerShell language mode can be found [here](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_language_modes?source=recommendations&view=powershell-7.3).  
+
+        ![PowerShell Version](../Images/00_PS_Language_Mode.png)  
+    
+  <br/>
+
+  4. Bicep installation:  
+
+      To run bicep based deployment scripts as part of infra setup bicep installation is required. Verify if bicep is installed by running below and install if not already available:
```

Comment:
To run bicep based deployment scripts as part of infra setup, a bicep installation is required. Verify if bicep is installed by running below and install if not already available:

Reply:
Addressed.